.github/workflows/test.yml

```diff
@@ -0,0 +1,49 @@
+name: Test
+on:
+  # Ensure GitHub actions are not run twice for same commits
+  push:
+    branches: [master]
+    tags: ['*']
+  pull_request:
+    types: [opened, synchronize, reopened]
```

Comment:
on traffic mesh we are not seeing duplicated runs with just `on: push`: https://github.com/netlify/traffic-mesh/blob/master/.github/workflows/rust.yml#L3

Reply:
I can confess that I copy&pasted from https://github.com/netlify/js-client/blob/1000eac0fdf9434336adefdde5cce1d46c02d60e/.github/workflows/workflow.yml#L2-L8 , any opinions/suggestions @ehmicky ? (as I copy pasted from you :P)

Reply:
This was suggested by @erezrokah and was originally copied from `netlify-cms`.
When doing only `push`, PRs that come from forked repositories do not seem to be included.
When doing both `push` and `pull_request`, PRs that come from non-forked repositories are run twice, which is the reason behind `branches: [master]`. The `tags` and `types` are just the default values (can be removed).

@erezrokah Please correct me if I'm wrong.

Reply:
ok, that explains why i haven't seen it on a private repo 😄

Reply:
nice 🎉 very good to know, thanks for tips @ehmicky and @erezrokah , sounds like we could copypasta this whenever we migrate other open source repos too

Reply:
Another thing to consider is that GitHub secrets are not passed to actions triggered from forked repositories.
We had to create a separate testing flow for the CMS project:
https://github.com/netlify/netlify-cms/blob/b5a242ec8ed627e7f5e2ce8af454dd1dda1f71cc/.github/workflows/nodejs.yml#L104

Reply:
good to know, thanks for sharing @erezrokah ! I think it won't be a problem with this repo but I'll keep it in my mind 👍